lib/ret/storage.ex

```diff
@@ -350,4 +350,19 @@ defmodule Ret.Storage do
       owned_file
     end)
   end
+
+  def in_quota?() do
+    with storage_path when is_binary(storage_path) <- module_config(:storage_path),
+         quota_gb when is_integer(quota_gb) and quota_gb > 0 <- module_config(:quota_gb) do
```

Comment:
Is this going to be configurable via ita? I wonder if we should make this configurable with the app config.

Reply:
yup, it's part of the stack configs

Reply:
https://github.com/mozilla/hubs-ops/blob/master/cloudformation/stack.yaml#L139